<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
   <head>
   <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

   <title>RECOMB 2010</title>
   <script src="http://www.google.com/jsapi?key=ABQIAAAA3Mb3buLF_NHBWQzevVJV2BTOd5Vsqkfq_MPuRWBSq1vAwZeVhRQoD_OUGVfFOSCPn_UHK5J91clW1w" type="text/javascript"></script>
   <script language="javascript" src="http://ajax.googleapis.com/ajax/libs/jquery/1.3.2/jquery.min.js" type="text/javascript"></script>
   
   <script language="javascript" src="jquery.tweet.js" type="text/javascript"></script> 
   <script type='text/javascript'>

		$(document).ready(function(){
		
    		$("#dim").css("height", $(document).height());
    		
    		$(".close").click(function(){
    			$("#dim").fadeOut();
    			return false;
			});
			
		
		});
		
		$(window).bind("resize", function(){
		 	$("#dim").css("height", $(window).height());
		});

		$(window).load(function () {
		var isloaded=0
		if (document.cookie.indexOf("isloaded")==-1){
		// run code
		$("#dim").fadeIn();
		document.cookie="isloaded=1"
		return false;
		}
		});

     
   $(document).ready(function(){
       $(".tweet").tweet({
	 username: "recomb2010",
	     join_text: "auto",
	     avatar_size: 32,
	     count: 2,
	     loading_text: "loading tweets..."
	     });
     });
</script> 

<script type = "text/javascript" src = "jquery.cyclic-fade.js"></script>
<script type = "text/javascript" >
	

	$(document).ready(function() {

		// default behavior
		<!-- $(".blink").cyclicFade(); -->
		$(".blink").cyclicFade({
			repeat: 0,
			params: [
				{fadeout:100, stayout:300, opout:0, fadein:100, stayin:2000, opin:1.0},
				{fadeout:100, stayout:300, opout:0, fadein:100, stayin:300, opin:0.75},
				{fadeout:100, stayout:300, opout:0, fadein:100, stayin:300, opin:1.0}
			]
		});
	});
	
	</script>

<link href="jquery.tweet.css" media="all" rel="stylesheet" type="text/css"/>
   <link rel="stylesheet" type="text/css" href="style.css" media="all">
   </head>
   <body leftmargin="0" topmargin="0" bgcolor="#bbbbbb" marginheight="0" marginwidth="0">
   <table align="center" bgcolor="#ffffff" border="0" cellpadding="0" cellspacing="0" width="800">
   <tbody>
   <tr>
   <td width="600"><a name="top" id="top"><img src="logo.jpg" usemap="#logomap" alt="RECOMB 2010 Logo" border="0" height="250" width="800"></a></td>
   <map id="logomap" name="logomap">

   <area shape="rect" 
   coords="79,219,119,229" 
   alt="Home"
   href="home.html">
   <area shape="rect" 
   coords="139,219,197,229" 
   alt="Program"
   href="program.html">

   <area shape="rect" 
   coords="218,219,261,229" 
   alt="Travel"
   href="travel_information.html">
   <area shape="rect" 
   coords="280,219,332,229" 
   alt="Authors"
   href="authors.html">
   <area shape="rect" 
   coords="352,219,433,229" 
   alt="Registration"
   href="http://www.esviagens.com/dmc/cong/recomb2010/registration-form_GB.html">
   <area shape="rect" 
   coords="453,219,534,229" 
   alt="Committees"
   href="committees.html">
   <area shape="rect" 
   coords="554,219,606,229" 
   alt="Contact"
   href="contact.html">
   <area shape="rect" 
   coords="627,219,736,229" 
   alt="Accommodation"
   href="http://www.esviagens.com/dmc/cong/recomb2010/index.html">
   </map>
   </tr>


   <!-- <div id="dim"> -->
   <!--   <div class="msgbox"> -->
   <!--     <a class="close" href="#" ><img src="close.jpg"/></a> -->
   <!--     <div class="regulartext">	     -->
   <!-- 	 <h3>Conference rescheduling notice</h3> -->
   <!-- 	 <p>Tuesday, April 20<span class="super">th</span>, 11:59PM</p>				     -->
   <!-- 	 <p>The date of the RECOMB-2010 conference was moved to <b>August 12-15<span class="super">th</span></b>.</p> -->
   <!-- 	 <p>Even though Lisbon airport remains open, many people will be facing significant difficulties -->
   <!-- 	 to travel here, because of connections in central Europe. Many more will probably -->
   <!-- 	 decide to cancel the trip, independently of the airport situation, because of all the -->
   <!-- 	 uncertainty this phenomenon is causing. We do firmly believe that the conditions to -->
   <!-- 	 organize a successful RECOMB meeting are not in place.</p> -->
   <!-- 	 <p>The organizing committees have therefore decided that the RECOMB-2010 conference -->
   <!-- 	 will be rescheduled to the dates of August 12-15<span class="super">th</span>, in Lisbon. All the invited -->
   <!-- 	 speakers and panelists have agreed to the change, and we expect that all attendees -->
   <!-- 	 will be able to make the necessary changes in travel arrangements. -->
   <!-- 	 </p> -->
   <!-- 	 <p>Your registration fee will, off course, be valid to the changed venue. -->
   <!-- 	 The organizing committee will take all steps to minimize the inconvenience that -->
   <!-- 	 results from this change, including the re-booking of the hotels and other reservations -->
   <!-- 	 you may have made with us.  As you will understand, these are difficult times, and this -->
   <!-- 	 was a very difficult decision. -->
   <!-- 	 </p> -->
   <!-- 	 <p>All requests for cancellation of registration and rebooking or cancellation -->
   <!-- 	 of the hotels should be sent directly to <a href="mailto:lisboa.congress@topatlantico.pt">lisboa.congress@topatlantico.pt</a>. -->
   <!-- 	 </p> -->
	 
   <!--     </div> -->
   <!--   </div> -->
   <!-- </div> --><tr height="600" valign="top">
  <?php include("./program_sidebar.php"); ?>
  <td height="100%" valign="top" width="100%"><table border="0" cellpadding="0" cellspacing="0" width="100%">
    <tbody><tr>
      <td width="100%"><table border="0" cellpadding="5" cellspacing="0">
	<tbody><tr>
	  <td><img src="transp.gif" height="5" width="1"></td>
	</tr>
	<tr>
	  <td class="menusubtopicheaderfont">RECOMB 2010 Accepted Papers</td>
	</tr>

	<tr>
	  <td><div class="regulartext">



	    <table align="center" width="500" border="0" cellspacing="0" cellpadding="0">
	      <tr>
		<td align="center"><table width="440" border="0" cellpadding="0" cellspacing="0">
		  <tr>
		    <td>&nbsp;</td>
		  </tr>
		  <tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center"><b>#</b></div></td>
			<td width="400"><div align="center"><b>Authors, Title</b></div></td>
		      </tr>
		    </table></td>
		  </tr>


<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">1</div></td>
			<td width="400"><div align="center">Nir Atias and Roded Sharan. An Algorithmic Framework for Predicting Side-Effects of Drugs</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">2</div></td>
			<td width="400"><div align="center">Ferhat Ay and Tamer Kahveci. SubMAP: Aligning metabolic pathways with subnetwork mappings</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">3</div></td>
			<td width="400"><div align="center">Sivan Bercovici, Itai Sharon, Ron Pinter and Tomer Shlomi. Pathway-Based Functional Analysis of Metagenomes</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">4</div></td>
			<td width="400"><div align="center">Sivan Bercovici and Dan Geiger. Admixture Aberration Analysis: Application to mapping in admixed population using pooled DNA</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">5</div></td>
			<td width="400"><div align="center">José Caldas and Samuel Kaski. Hierarchical Generative Biclustering for MicroRNA Expression Analysis</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">6</div></td>
			<td width="400"><div align="center">Salim A. Chowdhury, Rod K. Nibbe, Mark R. Chance and Mehmet Koyuturk. Subnetwork State Functions Define Dysregulated Subnetworks in Cancer</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">7</div></td>
			<td width="400"><div align="center">Manfred Claassen, Ruedi Aebersold and Joachim M. Buhmann. Proteome Coverage Prediction for Integrated Proteomics Datasets</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">8</div></td>
			<td width="400"><div align="center">Timothy Danford, Robin Dowell and David Gifford. Discovering Regulatory Overlapping RNA Transcripts</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">9</div></td>
			<td width="400"><div align="center">Constantinos Daskalakis and Sebastien Roch. Alignment-Free Phylogenetic Reconstruction</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">10</div></td>
			<td width="400"><div align="center">Jianxing Feng, Wei Li and Tao Jiang. Inference of Isoforms from Short Sequence Reads</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">11</div></td>
			<td width="400"><div align="center">Bjarni Halldorsson, Derek Aguiar, Ryan Tarpine and Sorin Istrail. The Clark Phase-able Sample Size Problem: Long-range Phasing and Loss of Heterozygosity in GWAS</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">12</div></td>
			<td width="400"><div align="center">Michael Hirsch, Bernhard Schoelkopf and Michael Habeck. A new algorithm for improving the resolution of cryo-EM density maps</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">13</div></td>
			<td width="400"><div align="center">Richard Jang, Xin Gao and Ming Li. Towards Automated Structure-based NMR Assignment</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">14</div></td>
			<td width="400"><div align="center">Kyowon Jeong, Sangtae Kim, Nuno Bandeira and Pavel Pevzner. Gapped Spectral Dictionaries and Their Applications for Databases Searches of Tandem Mass Spectra</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">15</div></td>
			<td width="400"><div align="center">Wei-Chun Kao and Yun S. Song. naiveBayesCall: An efficient model-based base-calling algorithm for high-throughput sequencing</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">16</div></td>
			<td width="400"><div align="center">David Kelley and Carl Kingsford. Extracting between-pathway models from E-MAP interactions using expected graph compression</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">17</div></td>
			<td width="400"><div align="center">Yoo-Ah Kim, Stefan Wuchty and Teresa Przytycka. Simultaneous Identification of Causal Genes and Dys-regulated Pathways in Complex Diseases</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">18</div></td>
			<td width="400"><div align="center">Geoffrey Koh, David Hsu and P. S. Thiagarajan. Incremental Signaling Pathway Modeling by Data Integration</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">19</div></td>
			<td width="400"><div align="center">Adam Kowalczyk, Justin Bedo, Thomas Conway and Bryan Beresford-Smith. Poisson Margin Test for Normalisation Free Significance Analysis of NGS Data</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">20</div></td>
			<td width="400"><div align="center">Christos Kozanitis, Chris Saunders, Semyon Kruglyak, Vineet Bafna and George Varghese. Compressing genomic sequence fragments using SlimGene</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">21</div></td>
			<td width="400"><div align="center">Fumei Lam, Charles H. Langley and Yun S. Song. On the Genealogy of Asexual Diploids</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">22</div></td>
			<td width="400"><div align="center">Jonathan Laserson, Vladimir Jojic and Daphne Koller. Genovo: De Novo Assembly For Metagenomics</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">23</div></td>
			<td width="400"><div align="center">Seunghak Lee, Eric Xing and Michael Brudno. MoGUL: Detecting Common Insertions and Deletions in a Population</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">24</div></td>
			<td width="400"><div align="center">Navodit Misra, Guy Blelloch, R Ravi and Russell Schwartz. Generalized Buneman pruning for inferring the most parsimonious multi-state phylogeny</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">25</div></td>
			<td width="400"><div align="center">Laurent Noé, Marta Gîrdea and Gregory Kucherov. Seed design framework for mapping AB SOLiD reads</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">26</div></td>
			<td width="400"><div align="center">Bogdan Pasaniuc, Noah Zaitlen and Eran Halperin. Accurate estimation of expression levels of homologous genes in RNA-seq experiments</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">27</div></td>
			<td width="400"><div align="center">Benedict Paten, Mark Diekhans, Jian Ma, Bernard Suh and David Haussler. Cactus Graphs for Genome Comparisons</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">28</div></td>
			<td width="400"><div align="center">Yu Peng, Henry C.M. Leung, SM Yiu and Francis Chin. IDBA - A Practical Iterative de Bruijn Graph De Novo Assembler</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">29</div></td>
			<td width="400"><div align="center">Sheila Reynolds, Zhiping Weng, Jeff Bilmes and William Stafford Noble. Predicting nucleosome positioning using multiple evidence tracks</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">30</div></td>
			<td width="400"><div align="center">Barna Saha, Allison Hoch, Samir Khuller, Louiqa Raschid and Xiao-Ning Zhang. Dense Subgraphs with Restrictions and Applications to Gene Annotation Graphs</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">31</div></td>
			<td width="400"><div align="center">Raheleh Salari, Mathias Mohl, Sebastian Will, S. Cenk Sahinalp and Rolf Backofen. Time and space efficient RNA-RNA interaction prediction via sparse folding</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">32</div></td>
			<td width="400"><div align="center">Manu Setty, Alexander Gusev and Itsik Pe'er. HLA type inference via haplotypes identical by descent</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">33</div></td>
			<td width="400"><div align="center">Fabio Vandin, Eli Upfal and Benjamin Raphael. Algorithms for Detecting Significantly Mutated Pathways in Cancer</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">34</div></td>
			<td width="400"><div align="center">Christian Widmer, Jose Miguel Leiva-Murillo, Yasemin Altun and Gunnar Rätsch. Leveraging Sequence Classification by Taxonomy-based Multitask Learning</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">35</div></td>
			<td width="400"><div align="center">Yu-Wei Wu and Yuzhen Ye. A novel abundance-based algorithm for binning metagenomic sequences using l-tuples</div></td>
		      </tr>
		    </table></td>
		  </tr>
<tr>
		    <td><table width="440" border="1" cellpadding="3" cellspacing="0" >
		      <tr>
			<td><div align="center">36</div></td>
			<td width="400"><div align="center">Jianyang Zeng, Pei Zhou and Bruce Donald. A Markov Random Field Framework for Protein Side-Chain Resonance Assignment</div></td>
		      </tr>
		    </table></td>
		  </tr>

		  <tr>
		    <td>&nbsp;</td>
		  </tr>
		</table>
		</td>
	      </tr>
	    </table>


	  </div></td>
	</tr>


      </tbody></table></td>
      <td><img src="transp.gif" height="1" width="15"></td>
    </tr>
  </tbody></table></td>
</tr>
</tbody></table></td>
</tr>
<tr>
  <td bgcolor="white"><img src="transp.gif" height="4" width="15">
  </td>
</tr>
<tr>
  <td bgcolor="black"><img src="transp.gif" height="2" width="15">
  </td>
</tr>
</tbody></table>
</body></html>
